pkg/packetbrokeragent/grpc_pba_test.go

```diff
@@ -232,17 +232,7 @@ func TestPba(t *testing.T) {
 						Name: &pbtypes.StringValue{
 							Value: "Test Network",
 						},
-						DevAddrBlocks: &iampb.DevAddrBlocksValue{
-							Value: []*packetbroker.DevAddrBlock{
-								{
-									Prefix: &packetbroker.DevAddrPrefix{
-										Value:  0x26000000,
-										Length: 24,
-									},
-									HomeNetworkClusterId: "test-cluster",
-								},
-							},
-						},
+						// NOTE: DevAddrBlocks are not updated here, as the tenant cannot change their own DevAddr blocks.
```

Comment:
Is this true ?

https://github.com/TheThingsIndustries/lorawan-stack/blob/575a3a4d4c78cc94c4fa68d95a5f5affe0de34e6/pkg/identityserver/tenant_registry.tti.go#L219-L227
https://github.com/TheThingsIndustries/lorawan-stack/blob/575a3a4d4c78cc94c4fa68d95a5f5affe0de34e6/pkg/identityserver/tenant_access.tti.go#L54-L56
https://github.com/TheThingsIndustries/lorawan-stack/blob/575a3a4d4c78cc94c4fa68d95a5f5affe0de34e6/pkg/identityserver/tenant_access.tti.go#L110-L112

Reply:
The tenant cannot change their own DevAddr blocks in Packet Broker. That is what is being tested here.

If tenants would be able to do that, they would be allowed to give themselves any DevAddr block and receive any traffic.